extensions/amp-bind/0.1/bind-validator.js

```diff
@@ -246,6 +246,7 @@ function createElementRules_() {
       'state': null,
     },
     'AMP-SELECTOR': {
+      'disabled': null,
```

Comment:
just a question. Why do wrap these properties in single quotes?

Reply:
Future-proofs in case we do aggressive property renaming with Closure Compiler: https://developers.google.com/closure/compiler/docs/limitations#implications-of-global-variable-function-and-property-renaming

Reply:
cool. thanks